### ISSUE TITLE
Request for Model checkpoints

### ISSUE THREAD

**Issue by Pratyush2wayne** [Yoonwoo-Ha/RACE-6D#1]
Hi,
Congratulations for the great work.
Can you please release the model checkpoints.

**Comment by Yoonwoo-Ha** [comment]
Hi, thanks for your patience!

We have released inference-only EMA checkpoints for LM-O, YCB-V, T-LESS (RGB and RGB-D), TUD-L, HB, and IC-BIN.

Download links, matching configurations, BOP AR scores, and SHA-256 checksums are available in the [Pretrained Checkpoints section of the README](https://github.com/Yoonwoo-Ha/RACE-6D#-pretrained-checkpoints).

The checkpoints can be evaluated as follows:

```bash
python tools/train.py \
    -c configs/race6d/r50vd/race6d_r50vd_lmo_rgb.yml \
    --test-only -r path/to/race6d_r50vd_lmo_rgb_ema.pth
```

These files contain inference EMA weights only. They can also be used for fine-tuning with `-t`, but do not include optimizer or scheduler state for exact training resumption.

The ITODD checkpoint is pending re-evaluation on the official BOP server and will be added separately.
